x-pack/plugins/security_solution/public/cases/components/case_view/index.tsx

```diff
@@ -162,6 +163,14 @@ export const CaseView = React.memo(({ caseId, subCaseId, userCanCrud }: Props) =
     [dispatch]
   );
 
+  const endpointDetailsHref = (endpointId: string) =>
```

Comment:
You should use `useCallback()` to define this function

Reply:
i do a useCallback when i actually use it in `plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.tsx` would i need to do it twice?